src/DetailView/DetailOKRList.tsx

```diff
@@ -16,22 +17,28 @@ export class DetailOKRList extends React.Component<IDetailOKRListProps, {}> {
             this.loadData(this.props.area.AreaId);
         }
     }
-    
+
     public componentDidUpdate(prevProps: IDetailOKRListProps) {
         if (this.props.area.AreaId !== prevProps.area.AreaId) {
             this.loadData(this.props.area.AreaId);
         }
     }
-    
+
     public render(): JSX.Element {
-        const [{ objectives }] = this.context;
-        return (<>
-            {objectives.map((objective, index) => <DetailOKR objective={objective} key={"DetailOKR" + index} />)}
-        </>);
+        const [{ objectives, selectedArea }] = this.context;
+        if (objectives && objectives.length > 0) {
+            return (<>
+                {objectives.map((objective, index) => <DetailOKR objective={objective} key={"DetailOKR" + index} />)}
+            </>);
+        }
+        else {
+            const areaName = selectedArea ? selectedArea.Name : "";
+            return <ObjectiveZeroData areaName={areaName} />
+        }
     }
 
     private loadData = (area: string) => {
-        const [{}, actions] = this.context;
-        actions.getObjectives({area: area})
+        const [{ }, actions] = this.context;
```

Comment:
@Nianwens The objectives here are getting reset to only this area's objectives. I thought the objectives were always all of them. Can I change this? What is your expectation?